src/RefreshToken.js

```diff
@@ -1,9 +1,13 @@
+import Environment from "./Components/Environment";
+
 function RefreshToken(fetchResults) {
+    var env = new Environment();
     if ("code" in fetchResults && fetchResults.code === "token_not_valid") {
-        var url = "https://recess-api.herokuapp.com/api/token/refresh/";
```

Comment:
really good catch on this

Reply:
thanks was an improvement of opportunity